vllm/engine/arg_utils.py

```diff
@@ -660,6 +661,14 @@ def add_cli_args(parser: FlexibleArgumentParser) -> FlexibleArgumentParser:
             type=str,
             default=None,
             help='Target URL to which OpenTelemetry traces will be sent.')
+        parser.add_argument(
+            '--collect-model-forward-time',
```

Comment:
what about something like 

--trace-components="model,scheduler," (it can be "all")

Reply:
The reason to put only this behind a flag is that it is possibly costly, with the use of a synchronize operation underneath. (fwiw, in our benchmarks it is not a factor). The others scheduler etc are costly. Reporting them to the traces gives more detail at no cost. 

What do you think ?

Reply:
I was thinking more of the case to support logging this (it is useful for dev). But I think as long as it is just for tracing, I have no problem with that